analyzers/tests/SonarAnalyzer.UnitTest/Rules/TooManyGenericParametersTest.cs

```diff
@@ -27,23 +27,26 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class TooManyGenericParametersTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<TooManyGenericParameters>();
         [TestMethod]
         public void TooManyGenericParameters_DefaultValues() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\TooManyGenericParameters_DefaultValues.cs", new TooManyGenericParameters());
+            builder.AddPaths("TooManyGenericParameters_DefaultValues.cs").Verify();
```

Comment:
Please rename the files, we tend to use `.` for new stuff
```suggestion
            builder.AddPaths("TooManyGenericParameters.DefaultValues.cs").Verify();
```

Reply:
The `DefaultValues` file has still underscore, only `CustomValues` was fixed.